drivers/periph_common/Kconfig.gpio

```
@@ -43,6 +47,11 @@ config MODULE_PERIPH_INIT_GPIO_FAST_READ
     default y if MODULE_PERIPH_INIT
     depends on MODULE_PERIPH_GPIO_FAST_READ
 
+config MODULE_PERIPH_INIT_GPIO_TAMPER_WAKE
+    bool "Auto initialize GPIO tamper wakeup pin"
```

```suggestion
    bool "Auto initialize GPIO tamper detection"
```
no pin is configured yet here